src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/JsonLogin/config.yml

```diff
@@ -16,7 +16,7 @@ security:
             pattern: ^/
             anonymous: true
             json_login:
-               check_path:    /chk
+               check_path: /chk
```

Comment:
@ronfroy If this has to be changed, it must be done on the lowest branch where it exists to prevent merge conflicts (but I don't think it's worth it, we should just keep it in mind for new code). Can you please revert this change? Same for the `custom_handlers.yml` file below